### DISCUSSION TITLE
How to close  v2 permissions all limit?

### DISCUSSION THREAD

**Discussion by A25468451** [tauri-apps/tauri#11832] [Ideas]
Oh, my friend. Permissions are very numerous and detailed, my application is very simple, I am not very familiar with tauri, I just want to focus on development. Can I turn off all permissions limit ?

**Comment by FabianLars** [comment]
> Can I turn off all permissions limit ?

No.

**Reply by A25468451** [reply]
Thank you for your reply. Many applications don't require such fine-grained permission restrictions, yet they have to focus their efforts on this. So perhaps we could add an 'enable and off' switch, instead of forcing everyone to use it.

**Reply by FabianLars** [reply]
I'm sorry but the team has already decided against this. Maybe they/we will re-consider at some point - i doubt it but only time will tell.

**Reply by A25468451** [reply]
That's okay. Thank you for your team's project. This project is amazing
